source/docs/hardware/hardware-basics/robot-battery.rst

```diff
@@ -0,0 +1,177 @@
+.. include:: <isonum.txt>
+
+Robot Battery
+=============
+
+The power supply for an FRC\ |reg| robot is a single 12V 18Ah SLA [Sealed Lead Acid] AGM [Absorbent Glass Mat] battery, capable of high momentary currents in excess of 180A when fully charged. The Robot Battery assembly includes the COTS battery, lead cables with contacts, and SB connector.
+
+Battery Safety & Handling
+-------------------------
+
+**DO NOT CARRY A BATTERY BY THE CABLES**, and always avoid pulling by them.
+Pulling on batteries by the cables may loosen, bend, or break the lug or battery tab -- or even tear the entire tab out of the housing!
+
+.. image:: images/robot-battery/broken-terminal.png
+  :alt: One terminal of an FRC battery fully detached from the battery.
+
+Even if it doesn't get broken, the battery will not be able to provide the same amount of current if the :ref:`connectors are loose <docs/hardware/hardware-basics/preemptive-troubleshooting:Check Battery Connections>`.
+
+Dropping the batteries can bend the internal plates, create bulges, or in the worst case - crack the battery case open. While we do use AGM [absorbant glass mat] technology for safety, when a cell is punctured it still will leak some liquid. This is why FIRST recommends teams have a battery spill kit available.
```

Comment:
Fixed AGM & modified to "A small amount of battery acid"? 

"May leak hazardous chemicals" strikes me as more sinister and open ended than a bit of battery acid. Not clear why it would be gelled - the MK ES17-12 datasheet only says nonspillable AGM, implying that they have a liquid infused glass mat that would leak liquid acid, but anecdotes indicate gel type extcretions from some legal batteries? 

At the same time our high impact physical use case implies the use of gel technology for better anti-vibration performance , but if it's not on the datasheet they're probably not spending those pennies on us. Maybe there's a mix of AGM and gel type in circulation.